utils/utils.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import "strings"
+
+/*
+Redact masks the given token by replacing part of the string with *
+*/
```

Comment:
For consistency with the rest of the codebase, would you mind using `//`-style comments?